### PR TITLE
Change index.html, style.css, script.js and added Folder "Images"

### PR DESCRIPTION
**Pull request by sroblesher**
Add "Images" folder with an image "Me.jpg" inside it.

Modify background color in style.css

Change function to display random fact in script.js.

Add image of me and a paragraph telling something about me in index.html.

Change style insertion of width and height on image in index.html.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,13 @@
...
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Salvador's Portfolio</h1>
+      <p>This is Salvador's portfolio.</p>
+      <p>I'm currently a STEP intern at Google from The University of Texas at El Paso.</p>
+      <img src = "/Images/Me.jpg" class = "myPortrait" \>
```
Comment (wait321) [inline portfolio/src/main/webapp/index.html:14]: The slash is going the wrong way. It's suppose to be `/>`. You can use this to validate your HTML https://www.freeformatter.com/html-validator.html

File: portfolio/pom.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (aminry) [inline portfolio/pom.xml:37]: nit: remove the todo

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,13 @@
...
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Salvador's Portfolio</h1>
+      <p>This is Salvador's portfolio.</p>
+      <p>I'm currently a STEP intern at Google from The University of Texas at El Paso.</p>
+      <img src = "/Images/Me.jpg" class = "myPortrait" />
+      <p>Click here to get a random fact about me:</p>
+      <button onclick="addRandomFact()">Push me!!</button>
+      <div id="randomFacts-container"></div>
     </div>
```
Comment (aminry) [inline portfolio/src/main/webapp/index.html:18]: use a consistent naming convention. In this case random-facts-container

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const randomFacts =
+      ['I like to code.', 'Soccer is my favorite sport.', 'My favorite color is blue.', 'I live in Mexico.'];
```
Comment (aminry) [inline portfolio/src/main/webapp/script.js:20]: nit: Please break the line to fit in 100 characters.
Each item in the array can be on a separate line.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -15,14 +15,14 @@
...
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const randomFact = randomFacts[Math.floor(Math.random() * randomFacts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factsContainer = document.getElementById('randomFacts-container');
```
Comment (aminry) [inline portfolio/src/main/webapp/script.js:26]: nit: No need for local variable if it is only used once.
document.getElementById('randomFacts-container').innerText = randomFact;

File: portfolio/pom.xml (hunk 2 above)
Comment (sroblesher) [reply portfolio/pom.xml:37]: Got it!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (sroblesher) [reply portfolio/src/main/webapp/index.html:18]: True!

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (sroblesher) [reply portfolio/src/main/webapp/script.js:20]: Ok, will do!

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (sroblesher) [reply portfolio/src/main/webapp/script.js:26]: Ok, will change!

**Pull request merged**